.github/workflows/os_x_mklbuild.yml

```diff
@@ -44,6 +44,7 @@ jobs:
 
       - name: Test project
         run: |
+          python -m pytest -n 4 --durations=50 --verbose tests/python/unittest/test_numpy_op.py -m 'not serial'
```

Comment:
Isn't test_numpy_op.py already included by the next line? Or is it just a temporary change to fail early on CI?

Reply:
Yes, mkl build will hang on some tests, so I make it fail early on CI.